Last-Update: 2013-08-25
Forwarded: no
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: don't try to install non-existent file (fixes FTBFS).

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -224,14 +224,14 @@
 
 install(FILES ${CMAKE_BINARY_DIR}/mysql-workbench.desktop DESTINATION ${WB_INSTALL_SHARED_DIR}/applications)
 install(FILES ${CMAKE_BINARY_DIR}/mysql-workbench.conf DESTINATION ${WB_INSTALL_ETC_DIR}/ld.so.conf.d)
 
-if (EXISTS COPYING)
+if (EXISTS "${CMAKE_SOURCE_DIR}/COPYING")
     install(FILES README COPYING DESTINATION ${WB_INSTALL_DOC_DIR})
     install(FILES build/debian/mysql-workbench-community.sharedmimeinfo DESTINATION ${WB_INSTALL_SHARED_DIR}/mime/packages RENAME mysql-workbench.xml)
     install(FILES build/debian/mysql-workbench-community.mime DESTINATION ${WB_INSTALL_SHARED_DIR}/mime-info RENAME mysql-workbench.mime)
 else ()
-    install(FILES README LICENSE.mysql DESTINATION ${WB_INSTALL_DOC_DIR})
+    install(FILES README DESTINATION ${WB_INSTALL_DOC_DIR})
     install(FILES build/debian/mysql-workbench-commercial.sharedmimeinfo DESTINATION ${WB_INSTALL_SHARED_DIR}/mime/packages RENAME mysql-workbench.xml)
     install(FILES build/debian/mysql-workbench-commercial.mime DESTINATION ${WB_INSTALL_SHARED_DIR}/mime-info RENAME mysql-workbench.mime)
 endif ()
 
